deepchecks/vision/checks/distribution/image_property_outliers.py

```diff
@@ -9,36 +9,26 @@
 # ----------------------------------------------------------------------------
 #
 """Module of ImagePropertyOutliers check."""
-import typing as t
-from collections import defaultdict
-from numbers import Number
 
 import numpy as np
 
-from deepchecks import CheckResult
-from deepchecks.core import DatasetKind
-from deepchecks.core.errors import DeepchecksProcessError, NotEnoughSamplesError
-from deepchecks.utils.outliers import iqr_outliers_range
-from deepchecks.utils.strings import format_number
-from deepchecks.vision import SingleDatasetCheck, Context, Batch
+from deepchecks.vision import Batch, VisionData
+from deepchecks.vision.checks.distribution.abstract_property_outliers import AbstractPropertyOutliers
 from deepchecks.vision.utils import image_properties
 
-
 __all__ = ['ImagePropertyOutliers']
 
-from deepchecks.vision.utils.image_functions import prepare_thumbnail
-
 
-class ImagePropertyOutliers(SingleDatasetCheck):
-    """Find outliers images with respect to the given image properties.
+class ImagePropertyOutliers(AbstractPropertyOutliers):
+    """Find outliers images with respect to the given properties.
 
-    The check computes several image properties and then computes the number of outliers for each property.
```

Comment:
why remove the word "image" here? It is image properties.

Reply:
it's kinda redundant from the name, don't you think? anyway added back